spec/models/piece_spec.rb

```diff
@@ -13,4 +15,40 @@
       expect(bishop.moved).to eq true
     end
   end
+
+  describe "capturable?" do
+    it "should return false if specified coordinates not occupied by any piece" do
+      white_pawn = FactoryGirl.create(:pawn, :white)
+      expect(white_pawn.capturable?(4,5)).to eq false
+    end
+
+    it "return false if specified coordinates occupied by same color piece" do
+      white_pawn = FactoryGirl.create(:pawn, :white, :x_position => 3, :y_position => 0, :captured => false)
+      other_white_pawn = FactoryGirl.create(:pawn, :white, :x_position => 4, :y_position => 1, :captured => false)
+
+      expect(other_white_pawn.capturable?(white_pawn.x_position,white_pawn.y_position)).to eq false
+    end
+
+    it "return true if specified coordinates occupied by opponent's piece" do
+      black_rook = FactoryGirl.create(:rook, :black, :x_position => 4, :y_position => 0, :captured => false)
+      white_pawn = FactoryGirl.create(:pawn, :white, :x_position => 4, :y_position => 1, :captured => false)
+
+
+      expect(white_pawn.capturable?(black_rook.x_position,black_rook.y_position)).to eq true
+    end
+  end
+
```

Comment:
Can't update attributes in the database for some reason, it works in the console.